composer.json

```
@@ -8,6 +8,7 @@
         "interface",
         "appveyor",
         "bamboo",
+        "buddy-works",
```

```suggestion
        "buddy",
```

It is the official name, I guess.